plugins/node/instrumentation-typeorm/src/types.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright The OpenTelemetry Authors, Aspecto
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { Span } from '@opentelemetry/api';
+import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+
+export enum ExtendedDatabaseAttribute {
+  DB_STATEMENT_PARAMETERS = 'db.statement.parameters',
```

Comment:
While this isn't a speced name similar to many others, there is still [this rule for semantic attributes](https://opentelemetry.io/docs/specs/semconv/general/attribute-naming/) which the existing name violates:

> Names SHOULD NOT coincide with namespaces. For example if service.instance.id is an attribute name then it is no longer valid to have an attribute named service.instance because service.instance is already a namespace

Since we already have `db.statement`, I think it is better to use the opportunity and rename it to `db.typeorm.parameters` (similar to `'db.postgresql.values'`, `'db.mysql.values'`). or some other good name that does not make `db.statement` a namespace.

Reply:
Changed it, thanks